lms/djangoapps/grades/tasks.py

```diff
@@ -23,50 +23,40 @@
 
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
-def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible):
+def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible, **kwargs):
```

Comment:
I did think about that.  But then, looking at the code, the code assumes and requires those parameters to be there.  The only reason we can possibly get away with putting `score_deleted` within `kwargs` is because the rest of the code is behind a feature flag.  However, long-term, I don't think the `kwargs` design pattern will work for aync tasks - unless the added parameter is truly an optional parameter.